src/api/app/instrumentations/workflow_step_instrumentation.rb

```diff
@@ -0,0 +1,16 @@
+module WorkflowStepInstrumentation
+  extend ActiveSupport::Concern
+
+  # Define callbacks with ActiveModel::Callback which is included in ActiveModel::Model
+  included do
+    define_model_callbacks :initialize
+
+    after_initialize :track_instantiation
+  end
+
+  private
+
+  def track_instantiation
+    RabbitmqBus.send_to_bus('metrics', "workflow_step,step=#{self.class.name.demodulize.underscore},action=instantiation count=1")
```

Comment:
With the workflow step class name being a tag, I will be able to group by `tag(step)`. I will create a Grafana panel for the instantiation of workflow steps without having to update this panel whenever a new step is introduced.